### PR TITLE
feat:  improve permission, account, and connection info on all screens.

### LINKED ISSUE #437: In Connect Apps screen, improve permission info
### Motivation

Currently it's showing a simple tooltip as a feedback of what permissions the connected apps have:
image

Could use same template of `Connection request` screen.
Have a blue info icon, and when user click, show a popup with something similar to (but changing title to: `Connected apps permissions:`):
image

### Usage example

_No response_

### Possible implementations

_No response_

### PR DESCRIPTION
**Pull request by tomiiide**
This PR contains a bunch of minor fixes and updates:
1. Improves the permission info on the connection screen
2. Updates the Signer Info to the new UI on the all Popup screens.
3. Updates the origin info to the new UI on all Popup screens.
4. Some code refactors.

closes #437, #483, #484 and #482

Demo:
https://www.loom.com/share/277d8d2f235c4164a4e0ddbb177433db

### CODE REVIEW

**Review by akharito1975 — APPROVED** [review]

**Review by luizstacio — CHANGES_REQUESTED** [review]
Can you remove the https:// and http:// it should just the domain origin.
Screenshot 2023-02-25 at 16 50 19

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
AccountInfo Loader has different height of AccountInfo component, and also loader elements are not quite in the same place

File: packages/app/src/systems/Core/components/PermissionCard/mocks/PERMISSIONS.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const PERMISSION_LIST = [
```
Comment (LuizAsFight) [inline packages/app/src/systems/Core/components/PermissionCard/mocks/PERMISSIONS.tsx:1]: can we rename this file to `permissions.tsx` ?

File: packages/app/src/systems/Settings/components/ConnectionEdit/ConnectionEdit.tsx [hunk 2]
```diff
@@ -23,12 +23,20 @@ export function ConnectionEdit({
   status,
   ...ctx
 }: ConnectionEditProps) {
+  if (!ctx.connection) return null;
+  const { origin, title, favIconUrl } = ctx.connection;
   return (
     <Layout.Content>
       <Stack gap="$3">
-        <Text leftIcon={Icon.is('Globe')} css={styles.title}>
+        {/* <Text leftIcon={Icon.is('Globe')} css={styles.title}>
```
Comment (LuizAsFight) [inline packages/app/src/systems/Settings/components/ConnectionEdit/ConnectionEdit.tsx:31]: can we remove the comment ?

**Review by LuizAsFight — CHANGES_REQUESTED** [review]

**Comment by tomiiide** [comment]
> AccountInfo Loader has different height of AccountInfo component, and also loader elements are not quite in the same place

Fixed, thanks for this.

**Review by luizstacio — CHANGES_REQUESTED** [review]
- On the signing screen, the header can be removed
Screenshot 2023-02-27 at 12 41 19

File: packages/app/playwright/crx/crx.test.ts [hunk 3]
```diff
@@ -338,8 +338,7 @@ test.describe('FuelWallet Extension', () => {
 
         // Confirm transaction
         await hasText(confirmTransactionPage, /0\.0000001.ETH/i);
-        await waitAriaLabel(confirmTransactionPage, senderAccount.name);
-        await getButtonByText(confirmTransactionPage, /confirm/i).click();
```
Comment (luizstacio) [inline packages/app/playwright/crx/crx.test.ts:342]: The sender account name is essential in this test to ensure it shows the expected account on the approval screen.

File: packages/app/src/systems/Core/components/AccountInfo/AccountInfo.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+export const Usage: StoryFn<AccountInfoProps> = (args) => (
+  <Box css={{ width: 300 }}>
+    <AccountInfo
+      {...args}
+      headerText={'Signer Account'}
+      account={MOCK_ACCOUNTS[0]}
+    />
+  </Box>
+);
```
Comment (luizstacio) [inline packages/app/src/systems/Core/components/AccountInfo/AccountInfo.stories.tsx:22]: We can improve here using a template and just changing the params argument. Take a look on ContentHeader story.

**Review by luizstacio — CHANGES_REQUESTED** [review]

**Comment by tomiiide** [comment]
> * On the signing screen, the header can be removed
> 
> Screenshot 2023-02-27 at 12 41 19

what header are you referring to?

The 'Signing a messsage to:' section, or the 'Signature Request' section?

File: packages/app/playwright/crx/crx.test.ts (hunk 3 above)
Comment (luizstacio) [reply packages/app/playwright/crx/crx.test.ts:342]: > But we don't show the account name on the approval screen, we only show the address. Screenshot 2023-02-27 at 4 27 36 PM Screenshot 2023-02-27 at 4 27 40 PM

The test search for a aria-label, with the name of the account you can add it to the Account Details component, or change the test to check by the address.

**Comment by LuizAsFight** [comment]
> * On the signing screen, the header can be removed
> 
> Screenshot 2023-02-27 at 12 41 19

it's not important to show current network? that's the only purpose of header now with new components... maybe we can think on a solution to show the network when needed, and remove header from all DAPP screens?

**Comment by LuizAsFight** [comment]
> > * On the signing screen, the header can be removed
> > 
> > Screenshot 2023-02-27 at 12 41 19
> 
> it's not important to show current network? that's the only purpose of header now with new components... maybe we can think on a solution to show the network when needed, and remove header from all DAPP screens?

cc @luizstacio

**Comment by LuizAsFight** [comment]
now `DAPP/Approve Transaction Screen` is the only screen that has the header.

I suggest we remove it, as:
- now we already use AccountInfo component showing screen title as header
- current Header only purpose is show the network
- it's a too discrete place to show the network
 
 
we should:
- remove header component
- show network in other place, adding more relevance to it in layout hierarchy 
  - could be a card between Confirm / TxFromTo


doesn't need to be done in this PR but we can open an issue to fix it later if you guys agree @luizstacio @tomiiide 
image

**Comment by tomiiide** [comment]
> Confirm block is looking to big:
> 
> * decrease font-size by 2px for yellow(title) and gray(subtitle)
> * decrease margin between title and subtitle
> 
> image

Screenshot 2023-03-01 at 12 10 22 PM
Is this better?

**Review by LuizAsFight — APPROVED** [review]

**Comment by LuizAsFight** [comment]
awesome work @tomiiide

**Review by luizstacio — APPROVED** [review]

**Pull request merged**